beacon/sync/src/main/java/tech/pegasys/teku/beacon/sync/historical/ReconstructHistoricalStatesService.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright ConsenSys Software Inc., 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.beacon.sync.historical;
+
+import org.apache.commons.lang3.NotImplementedException;
+import tech.pegasys.teku.infrastructure.async.SafeFuture;
+import tech.pegasys.teku.service.serviceutils.Service;
+
+public class ReconstructHistoricalStatesService extends Service {
+
+  @Override
+  protected SafeFuture<?> doStart() {
+    return SafeFuture.failedFuture(
+        new NotImplementedException(
+            "ReconstructHistoricalStatesService start has not been implemented"));
```

Comment:
I'd just return `SafeFuture.COMPLETE` here and in doStop().  That way it can be hooked in but safely does nothing rather than causing failures.